### DISCUSSION TITLE
DHCP config not displaying or updating from UI

### DISCUSSION THREAD

**Discussion by Bostonvex** [RaspAP/raspap-webgui#961] [Q&A] [answered]
I was using an older version of RASPAP until this morning when I upgraded to the latest release. Since upgrading, the DHCP config page is not pulling in any data from the associated config files and it is not updating it either. Consequently dhcp for AP clients aren't functioning.

I figured that I bungled something up and then did a reimage of the Pi and did a fresh install RASPAP. However, the issue persists. If I add config via the UI for WLAN0, the UI says it saves, but the config lines are blank post save. See the screen shot below.

I then manually added /etc/dnsmasq.d/090_wlan0.conf as it did not exist nor was it created during install.

# RaspAP wlan0 configuration
pi@raspberrypi:/etc/dnsmasq.d $ sudo cat 090_wlan0.conf
#RaspAP wlan0 configuration
interface=wlan0
dhcp-range=10.3.141.50,10.3.141.254,255.255.255.0,12m
dhcp-option=6,8.8.8.8
pi@raspberrypi:/etc/dnsmasq.d $

# Permissions
pi@raspberrypi:/etc/dnsmasq.d $ ls -la
total 20
drwxr-xr-x   2 root root 4096 Jul 13 13:17 .
drwxr-xr-x 120 root root 4096 Jul 13 13:34 ..
-rw-r--r--   1 root root   79 Jul 13 13:32 090_raspap.conf
-rw-r--r--   1 root root  121 Jul 13 13:32 090_wlan0.conf
-rw-r--r--   1 root root  211 Feb  5 06:24 README
pi@raspberrypi:/

Not sure if this is a permissions issue or not but would appreciate help and suggestions.

<img width="907" alt="dhcpconfigUI" src="https://user-images.githubusercontent.com/87387992/125500640-33f594e7-e9a8-4ce4-9799-cfc7f7dd35e1.PNG">

**Comment by billz** [comment, marked as answer]
Version 2.6 was a significant refactoring of RaspAP's internals, including DHCP.
https://github.com/RaspAP/raspap-webgui/releases/tag/2.6-beta

If your previous version was 2.5.x you will need to do a fresh install.

**Comment by Bostonvex** [comment]
Thanks much for helping me understand the changes in 2.6. I got it to work after reading the notes and switching from Chrome to Edge...weird.

What threw me for a loop was that there appears to be an issue when I use Chrome to access the UI. The fields do not populate on Chrome and I had to use  Edge. Screenshots below of experience when using different browsers.
*You can ignore the red X's as my iptables config blocks DNS to prevent DNS leakage while using VPN.

# Values populate on Edge
<img width="623" alt="Edge_pop" src="https://user-images.githubusercontent.com/87387992/125611204-ae6357fb-5809-45be-8bef-791e668db42d.PNG">
<img width="835" alt="Edge_pop_2" src="https://user-images.githubusercontent.com/87387992/125610960-79fdd93e-fcc2-45c5-8b9a-1088d8919ebc.PNG">

# Values do not populate on Chrome
<img width="697" alt="chrome_no_pop" src="https://user-images.githubusercontent.com/87387992/125610803-7fa9559f-1195-45a4-bf86-75153ad0a99f.PNG">
<img width="955" alt="chrome_no_pop2" src="https://user-images.githubusercontent.com/87387992/125610815-f2ce1996-d42d-4860-b31f-0b81e21bbede.PNG">

**Comment by billz** [comment]
Most likely Chrome is caching one or more JavaScript files used by RaspAP. Clearing your cache will usually fix this.

**Reply by Bostonvex** [reply]
That worked, thx.
